roxie/roxiemem/roxiemem.cpp

```diff
@@ -3415,8 +3415,8 @@ class BufferedRowCallbackManager
     Owned<BackgroundReleaseBufferThread> backgroundReleaseBuffersThread;
     Owned<ReleaseBufferThread> releaseBuffersThread;
     CRowManager * owner;
-    atomic_t releasingBuffers;  // boolean if pre-emptive releasing thread is active
-    atomic_t releaseSeq;
+    std::atomic_uint releasingBuffers;  // boolean if pre-emptive releasing thread is active
```

Comment:
Seems to be being initialized/used as a boolean...

Why is this one std::atomic_uint and the others atomic_uint (without the std:: ) ?